repo_refactoring/data/sanity_data/generate_data.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Oct 16 21:43:47 2017
+
+@author: changyaochen
+"""
+
+
+def generate_data(id_filename, save_filename, 
```

Comment:
Thanks for writing this, I sort of prefer the following signature
generate_data_from_ids(data_dir), and I think this piece of code should live in data_type.py

Reply:
done